aqueductcore/backend/services/extensions_executor.py

```diff
@@ -177,6 +180,19 @@ async def execute(
         if exp_parameter is None:
             raise AQDValidationError(f"Action {extension}:{action}" " has no experiment parameters")
 
+        query = (
+            select(orm.Experiment)
+            .options(joinedload(orm.Experiment.created_by_user))
+            .where(orm.Experiment.uuid == experiment_uuid)
+        )
+        cursor = await db_session.execute(query)
+        experiment = cursor.scalars().first()
+        if experiment is None:
+            raise AQDValidationError("Experiment not found in the database.")
+
+        if not user_info.can_create_task_in_experiment_owned_by(experiment.created_by_user.uuid):
+            raise AQDPermission("User has no permission to start tasks in this experiment.")
```

Comment:
As we're using a different name in the Figma
```suggestion
            raise AQDPermission("User has no permission to run a job in this experiment.")
```